sqlmesh/core/audit/__init__.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: F401
```

Comment:
why don't we just always ignore f401, what is this?

Reply:
We don't want to because it checks [unused imports](https://docs.astral.sh/ruff/rules/unused-import/#legend). The `__init__` files though export names, so they're not really "unused". We also [do this in sqlglot](https://github.com/tobymao/sqlglot/blob/main/sqlglot/__init__.py).

There may be an issue with _how_ we're exporting names. IIRC there was a mypy issue in the past that was resolved after doing `import foo as foo` vs `import foo`, so maybe adding these aliases will allow us to get rid of the `noqa`s.

Reply:
yea let's do import foo as foo right

Reply:
yep planning to do another pass to verify this & fix if it's the culprit